renderer/components/Pay/PayPanelBody.js

```diff
@@ -95,8 +99,9 @@ PayPanelBody.propTypes = {
   initialAmountFiat: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
   intl: intlShape.isRequired,
   invoice: PropTypes.object,
-  isLn: PropTypes.bool,
+  isBolt11: PropTypes.bool,
   isOnchain: PropTypes.bool,
+  isPubkey: PropTypes.bool,
```

Comment:
This is probably ripe to be replaced with ENUM. but that might be out of scope, so we can create a "refactor" issue for this